test/acceptance/features/bindAppToMultipleServices.feature

```diff
@@ -55,3 +55,56 @@ Feature: Bind a single application to multiple services
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-2" should be changed to "True"
         And "nodejs-app" deployment must contain SBR name "binding-request-1"
         And "nodejs-app" deployment must contain SBR name "binding-request-2"
+
+        Scenario: Bind two backend services by creating 1 SBR to a single application
```

Comment:
for maintaining consistency we can say `two backend services by creating one SBR` or ` 2 backend services by creating 1 SBR` . Former sounds better as we are maintaining it in our feature files.

Reply:
Also in our tests, we are mentioning binding an app to a service, instead of service to an app

Reply:
> for maintaining consistency we can say `two backend services by creating one SBR` or ` 2 backend services by creating 1 SBR` . Former sounds better as we are maintaining it in our feature files.

This name is consistent with the scenario already present in the file. https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/bindAppToMultipleServices.feature#L11

Reply:
@pratikjagrut - the scenario description look good.

@Avni-Sharma - `two backend services by creating one SBR` or `2 backend services by creating 1 SBR` misses a verb, i.e. action.